src/app/recipient/components/recipient-earned-badge-list/recipient-earned-badge-list.component.html

```diff
@@ -1,255 +1,220 @@
+						<section class="emptysection l-emptysection" *ngIf="allBadges.length == 0">
+							<h2 class="emptysection-x-headline">{{ 'RecBadge.noBadgesYet' | translate }}</h2>
+							<p class="emptysection-x-copy">{{ 'RecBadge.collectShareBadges' | translate }}</p>
+							<div class="emptysection-x-imagecontainer">
+								<img alt="Illustration of backpack with badges" class="emptysection-x-image"
+									[src]="noBadgesImageUrl" />
+							</div>
+						</section>
+
+						<!-- Badge Search, group and grid/list toggle -->
+						<div class="oeb-section-sm" *ngIf="allBadges.length > 0">
+							<div class="tw-flex tw-flex-wrap tw-items-center oeb-standard-padding-bottom">
+								<input type="text" name="forminput" id="forminput" placeholder="Badges durchsuchen"
+									[(ngModel)]="searchQuery"
+									class="tw-w-[300px] tw-border-solid tw-border-purple tw-bg-white tw-mr-6"
+									hlmInput />
+								<oeb-checkbox class="tw-mt-2 md:tw-mt-0" name="groupby" [(ngModel)]="groupByIssuer"
+									[text]="'RecBadge.groupByInstitution' | translate"></oeb-checkbox>
+							</div>
+
+							<!-- Grid View -->
+							<ng-template [ngIf]="badgesDisplay == 'grid'">
+								<!-- ------------------ UNGROUPED ------------------  -->
+								<ng-template [ngIf]="!groupByIssuer">
+									<div class="l-grid l-grid-large tw-gap-10">
+										<!-- TODO: Fix the badgecard link for sharing and routing -->
+										<bg-badgecard *ngFor="let badgeResult of badgeResults"
+											[badgeTitle]="badgeResult.badge.badgeClass.name"
+											[badgeImage]="badgeResult.badge.image"
+											[badgeIssueDate]="badgeResult?.badge?.issueDate"
+											[mostRelevantStatus]="badgeResult.badge.mostRelevantStatus"
+											[issuerTitle]="badgeResult.badge.badgeClass.issuer.name"
+											[issuerSlug]="badgeResult.badge.badgeClass.issuer.slug"
+											[badgeSlug]="badgeResult.badge.slug"
+											(shareClicked)="shareBadge(badgeResult.badge)"></bg-badgecard>
+									</div>
+								</ng-template>
+								<!-- ------------------ GROUP BY ISSUER ------------------  -->
+								<ng-template [ngIf]="groupByIssuer">
+									<div *ngFor="let issuerGroup of issuerResults">
+										<h3 class="u-text-h3-semibold u-margin-bottom2x u-margin-top6x u-text-dark1">
 											{{ issuerGroup.issuer.name }}
-										</a>
-									</td>
-									<td class="datatable-x-cell u-text-body">
-										<time [date]="badge?.issueDate" format="mediumDate"></time>
-									</td>
-									<td class="datatable-x-cell">
-										<button
-											class="u-text-link-small u-text-bold u-margin-leftauto"
-											(click)="shareBadge(badge)"
-										>
-											Teilen
-										</button>
-									</td>
-								</tr>
+											<span class="u-text-small-semibold-caps u-text-dark4 u-margin-left2x">{{
+												issuerGroup.badges.length }}
+												{{ issuerGroup.badges.length == 1 ? 'Badge' : 'Badges' }}</span>
```

Comment:
Would it make sense to also have this in a language resource file?

Reply:
since we only support German/English this is fine for now. If we want more languages yes.